sqldb/migrations.go

```diff
@@ -70,6 +70,11 @@ var (
 			// schema. This is optional and can be disabled by the
 			// user if necessary.
 		},
+		{
+			Name:          "000007_invoice_modification",
```

Comment:
This is supposed to add the index? Maybe the file is missing from the PR now as the tests fail.

Reply:
Maybe something as simple as `000007_invoices` if we do more changes to the schema than adding the index, otherwise `000007_invoices_settle_id_index`.